src/libmongoc/tests/json/transactions/retryable-commit.json

```diff
@@ -121,8 +123,166 @@
               },
               "startTransaction": null,
               "autocommit": false,
+              "writeConcern": {
```

Comment:
How come we expect to see three different commitTransaction commands in this test?